clients/fides-js/src/lib/initialize.ts

```diff
@@ -408,14 +410,37 @@ export const initialize = async ({
   const { fides_meta, identity, fides_string, tcf_consent } = fides.cookie;
 
   // used to set Fides.consent
-  const consent = applyOverridesToConsent(
+  const consentObject = applyOverridesToConsent(
     fides.cookie.consent,
     fides.experience?.non_applicable_privacy_notices,
     fides.experience?.privacy_notices,
     options.fidesConsentFlagType ?? undefined,
     options.fidesConsentNonApplicableFlagMode ?? undefined,
   );
 
+  const hasPrivacyNotices =
```

Comment:
After any consent action (save, reject, accept) the consent object goes back to being a POJO. Is that expected?

Reply:
I moved the Proxy logic to be part of the `applyOverridesToConsent` util to avoid this issue. We need to run consent through that every time already anyway.

Reply:
That wasn't the correct change. `applyOverridesToConsent` is used in places where this causes a problem. I reverted that change and created a new `createConsentProxy` util to use only when we need it.